galleries/examples/text_labels_and_annotations/rainbow_text.py

```diff
@@ -3,73 +3,32 @@
 Rainbow text
 ============
 
-The example shows how to string together several text objects.
-
-History
--------
-On the matplotlib-users list back in February 2012, Gökhan Sever asked the
-following question:
-
-  | Is there a way in matplotlib to partially specify the color of a string?
-  |
-  | Example:
-  |
-  | plt.ylabel("Today is cloudy.")
-  |
-  | How can I show "today" as red, "is" as green and "cloudy." as blue?
-  |
-  | Thanks.
-
-The solution below is modified from Paul Ivanov's original answer.
+The example strings together several Text objects with different properties
+(e.g., color or font), positioning each one after the other: each Text other
+than the first one created with `~.Axes.annotate` and positioned so that its
+lower left corner is at the lower right corner (``xy=(1, 0)``) of the previous
+one (``xycoords=text``).
```

Comment:
```suggestion
(e.g., color or font), positioning each one after the other: The first Text is
directly created using `~.Axes.text`. All subsequent Texts are created with
`~.Axes.annotate`, which allows positioning so that its lower left corner is at
the lower right corner (``xy=(1, 0)``) of the previous one (``xycoords=text``).
```